.husky/reuse

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if command -v reuse &> /dev/null
+then
+    (cd databrowser && reuse lint)
+elif command -v pipx &> /dev/null
```

Comment:
I know we had a discussion about Windows / Python and your solution looks good. But after rethinking it, I'm not sure if we should put the `pipx` check / invocation here.

The reason is, that this invocation would start to download / install stuff the first time without that a developer expects that to happen. Another problem could be, that the `pipx` invocation fails (e.g. the download fails because the user is behind a proxy). In that case the whole commit process may break.

I think it may be better to leave just the `reuse` check / invocation and to point the developer to some documentation that says what can be done to fix the "problem". For example for `pipx` invocation, one could define an alias / link / script to `pipx run reuse` with name `reuse` that can than be invoked by the pre-commit hook.

What is your opinion on this?